src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,18 @@
+.catalog {
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+
+  &__list {
+    width: 1000px;
```

Comment:
Checklist rule 13 don't hardcode max-width, your catalog container should work fine for any number of cards. Use variables instead hardcore width. Replace justify-content in reason, when you have 7,6,..., card your grid looks incorrectly.